packages/freetype.rb

```diff
@@ -31,4 +31,10 @@ class Freetype < Meson
   depends_on 'zlib' # R
 
   meson_options '-Dharfbuzz=enabled'
+
+  def self.postbuild
+    # Remove freetype files that conflict with the harfbuzz package.
```

Comment:
This is backwards.

The freetype package is built on top of harfbuzz, and the files from freetype supersede the files from harfbuzz. 

So you build one, then the other, then the other.

Harfbuzz requires this update dance:

**# build order: harfbuzz => freetype => fontconfig => cairo => pango**

Reply:
I did it in order.  The harfbuzz build requires the freetype2 files to succeed.